library/core/src/iter/range.rs

```diff
@@ -512,33 +508,87 @@ macro_rules! range_incl_exact_iter_impl {
     )*)
 }
 
-#[stable(feature = "rust1", since = "1.0.0")]
-impl<A: Step> Iterator for ops::Range<A> {
+/// Specialization implementations for `Range`.
+trait RangeIteratorImpl {
```

Comment:
Bikeshed: We don't have consistent naming for specialization helper traits yet. Some suffix it with `Impl` and some prefix it with `Spec`.

Reply:
What should be done for this? I don't have a preference either way.

Reply:
Fine as-is, I don't think consistency for internal traits matters that much.